toolkit/templates/forms/option-select.html

```diff
@@ -2,7 +2,7 @@
   <div class="container-head js-container-head">
     <div class="option-select-label">{{ label }}</div>
   </div>
-  <div class="options-container" id="{{ options_container_id }}">
+  <div class="options-container" id="{{ options_container_id }}" data-framework="{{ framework_family }}" data-current-lot="{% if current_lot %}{{ current_lot.slug }}{% endif %}">
```

Comment:
What if there's no `framework_family`? If an option select is used for something not framework related, you'll get something weird here?

Reply:
Good point, I'll get that updated.